packages/console/src/pages/ApplicationDetails/components/Permissions/index.module.scss

```diff
@@ -0,0 +1,12 @@
+@use '@/scss/underscore' as _;
+
+
+// Override the Table  component styles
+[class*='tableContainer'] [class*='bodyTable'] tr.sectionTitleRow {
```

Comment:
This looks hacky, and without the module name I'm afraid it would take effect globally and thus might cause unexpected results

Reply:
Could apply an extra parent layer limit the scope of this style

Reply:
updated